src/m365/tenant/commands/service/service-message-list.ts

```diff
@@ -0,0 +1,99 @@
+import auth from '../../../../Auth';
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import Command, { CommandOption } from '../../../../Command';
+import Utils from '../../../../Utils';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  workload: string;
+}
+
+class TenantServiceMessageListCommand extends Command {
+  public get name(): string {
+    return `${commands.TENANT_SERVICE_MESSAGE_LIST}`;
+  }
+
+  public get description(): string {
+    return 'Gets the service messages regarding services in Office 365 from the Office 365 Management API';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    if (this.verbose) {
+      cmd.log(`Getting the service messages regarding services in Office 365 from the Office 365 Management API`);
+    }
+
+    const serviceUrl: string = 'https://manage.office.com/api/v1.0';
+    const statusEndpoint: string = (typeof args.options.workload != 'undefined' && args.options.workload) ? `ServiceComms/Messages?$filter=Workload eq '${escape(args.options.workload)}'` : 'ServiceComms/Messages';
+    const tenantId: string = Utils.getTenantIdFromAccessToken(auth.service.accessTokens[auth.defaultResource].value);
+
+    const requestOptions: any = {
+      url: `${serviceUrl}/${tenantId}/${statusEndpoint}`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      json: true
+    };
+
+    request
+      .get(requestOptions)
+      .then((res: any): void => {
+        if (args.options.output === 'json') {
+          cmd.log(res);
+        }
+        else {
+          cmd.log(res.value.map((r: any) => {
+            return {
+              Workload: r.Workload,
```

Comment:
For message center messages `Workload` and `ImpactDescription` are empty. Instead, we need to read `AffectedWorkloadDisplayNames` and `Title`.

We should change the code accordingly:

```ts
return {
  Workload: r.Id.startsWith('MC') ? r.AffectedWorkloadDisplayNames.join(', ') : r.Workload,
  Id: r.Id,
  Message: r.Id.startsWith('MC') ? r.Title : r.ImpactDescription
}
```